blog: shaping inner voice (#14)
Nathen-Smith/blog: posts/inner-voice.md

I felt reborn going into winter break. Overwhelming clarity and feelings of being refreshed hit like waves as I left campus and got on the airplane for home. But what was I to do without needing to worry about school or cooking? Lucky for me, my subconscious made plans. Having spent a whole semester practically giving no time to simply sit down and be with my thoughts, my subconscious was neglected and gasping for air. In particular, it was dying to tell me that I was not fulfilled and was losing touch with former hobbies.

> While my subconscious thoughts briefly surfaced in the form of the occasional shower thought, I still had given no time to the inner voice. Why was this important? For one, the inner voice impacts how you view yourself and hear others. In _You're Not Listening_ by Kate Murphy, Murphy illustrates the impact of the inner voice by contrasting two vastly different voices:

Author: ```suggestion
While my subconscious thoughts briefly surfaced in the form of the occasional shower thought, I still had given no time to the inner voice. Why was this important? For one, the inner voice impacts how you view yourself and hear others. In _You're Not Listening_ by Kate Murphy, Murphy illustrates the impact of the inner voice by contrasting two voices:
```

> An inner voice that says, Are you sure you want to do that? Why don’t you put yourself in their shoes? And Yeah, that was hurtful, but maybe they didn’t intend to hurt you, is a very different voice from the one that says, They are all out to get me and I’m no good. The latter voice is the one that makes you react in ways that are not to your benefit (Murphy 2020, 131)

By internalizing the qualities of the second voice, one is more likely to become more bitter and self-loathing, which is only reinforced when social interactions are perceived as worse than they actually were. The subconscious becomes intrusive and erratic, and the inner voice becomes irrational: resulting in a cloudy and unhappy mind. On the other hand, internalizing the qualities of the first voice makes you more understanding and empathetic. Most importantly, you will feel at peace.

But how is the inner voice shaped? And how can we improve it? First, let's try to better define the inner voice. We know the inner voice has a main dialogue, but it also has the ability to talk to itself from other perspectives:

> > As believed by many great philosophers and social theorists, including William James, Charles Sanders Peirce, and George Herbert Mead, believed the self generates conversations with itself by taking the perspective of another. An example is an athlete who might internaize the voice of a coach. Or you might have a back-and-forth with yourself that resembles changes you've had with your mother, boss, spouse, sibling, friend, or maybe a therpiast (Murphy 2020, 129)

Author: ```suggestion
> As believed by many great philosophers and social theorists, including William James, Charles Sanders Peirce, and George Herbert Mead, believed the self generates conversations with itself by taking the perspective of another. An example is an athlete who might internaize the voice of a coach. Or you might have a back-and-forth with yourself that resembles changes you've had with your mother, boss, spouse, sibling, friend, or maybe a therapist (Murphy 2020, 129)
```

> > As believed by many great philosophers and social theorists, including William James, Charles Sanders Peirce, and George Herbert Mead, believed the self generates conversations with itself by taking the perspective of another. An example is an athlete who might internaize the voice of a coach. Or you might have a back-and-forth with yourself that resembles changes you've had with your mother, boss, spouse, sibling, friend, or maybe a therapist (Murphy 2020, 129)

Author: ```suggestion
> In his book _The Voices Within_, Fernyhough writes that many great philosophers and social theorists, including William James, Charles Sanders Peirce, and George Herbert Mead, believed the self generates conversations with itself by taking the perspective of another. An example is an athlete who might internaize the voice of a coach. Or you might have a back-and-forth with yourself that resembles exchanges you've had with your mother, boss, spouse, sibling, friend, or maybe a therapist (Murphy 2020, 129)
```

> Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (better understood as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is the minimum. To an extent, your default responses are your personality, so enable and encourage them to flourish.

Author: ```suggestion
Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (better understood as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is just one part of the puzzle. To an extent, your default responses are your personality, so enable and encourage them to flourish.
```

> Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (better understood as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is just one part of the puzzle. To an extent, your default responses are your personality, so enable and encourage them to flourish.

Author: ```suggestion
Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (better understood as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is just one part of the puzzle. To an extent, your default responses are your personality, as argued by founder of Healthy Gamer GG Dr.K, so enable and encourage them to flourish.
```

> Another way to shape the inner voice is by being an open and honest listener. The more people you listen to, "the more sides of the conversation the inner voice can take and the more solutions you can imagine" (Murphy 2020, 130). This isn't to say to listen to everyone; some people are not worth listening to. Rather, listening to the open-minded will allow their good traits to transpire to your inner voice, and your inner voice will start to generate more understanding and constructive viewpoints.

Author: ```suggestion
Another way to shape the inner voice is by being an open and honest listener. The more people you listen to, "the more sides of the conversation the inner voice can take and the more solutions you can imagine" (Murphy 2020, 130). This isn't to say to listen to everyone; some people aren't worth listening to. Rather, listening to the open-minded will allow their good traits to transpire to your inner voice, and your inner voice will start to generate more understanding and constructive viewpoints.
```

But how is the inner voice shaped? And how can we improve it? First, let's try to better define the inner voice. We know the inner voice has a main dialogue, but it also has the ability to talk to itself from other perspectives:

> In his book _The Voices Within_, Fernyhough writes that many great philosophers and social theorists, including William James, Charles Sanders Peirce, and George Herbert Mead, believed the self generates conversations with itself by taking the perspective of another. An example is an athlete who might internalize the voice of a coach. Or you might have a back-and-forth with yourself that resembles exchanges you've had with your mother, boss, spouse, sibling, friend, or maybe a therapist (Murphy 2020, 129)

> Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (better understood as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is just one part of the puzzle. To an extent, your default responses are your personality, so enable and encourage them to flourish.

Author: ```suggestion
Therefore, the inner voice's tone and viewpoints are shaped by experiences with other people—including indirect interactions like watching TV personalities or reading a book. While you can't exactly seek positive experiences in the future nor remove past experiences (also known as Samskara in Indian philosophy), you can change how you react. By focusing on your default response, problems become apparent and you become self-aware. Remember the unhelpful voice from the first quote block, They are all out to get me and I’m no good? On paper, it is easy to evaluate this reaction and recognize that it is irrational and defeating: people rarely have harmful intentions. However, focusing on fixing the negatives is just one part of the puzzle. To an extent, your default responses are your personality, so enable and encourage them to flourish.
```
